frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -530,3 +530,6 @@ export type QueryWithDescription = {
   query: string;
   desc: string;
 };
+
+export type UseFlag = (flag: string, value: boolean) => void;
```

Comment:
shouldn't this be without `value` param ? 
```suggestion
export type UseFlag = (flag: string) => void;
```